android/app/src/main/assets/config.json

```diff
@@ -1,5 +1,5 @@
 {
   "Email": "dev@fossasia.org",
   "App_Name": "Open Event",
-  "Api_Link": "https://eventyay.com/api/v1/events/6/"
+  "Api_Link": "https://api.eventyay.com/v1/events/69/"
```

Comment:
https://github.com/Aconex/drakov this is good. Tried it out. Also supports API Blueprint.